app/src/main/java/org/oppia/app/topic/review/reviewitemviewmodel/TopicReviewItemViewModel.kt

```diff
@@ -0,0 +1,7 @@
+package org.oppia.app.topic.review.reviewitemviewmodel
+
+import androidx.lifecycle.ViewModel
+import org.oppia.app.model.Subtopic
+import org.oppia.app.topic.review.ReviewSubtopicSelector
+
+class TopicReviewItemViewModel(val subtopic: Subtopic, val onReviewItemPressed: ReviewSubtopicSelector): ViewModel()
```

Comment:
Add KDoc for this class

Reply:
EOF is missing

Reply:
The indentation is incorrect, we there are multiple parameters write them in multiple lines:
```
class TopicReviewItemViewModel(
  val subtopic: Subtopic,
  val onReviewItemPressed: ReviewSubtopicSelector
) : ViewModel()
```